src/config.rs

```diff
@@ -24,6 +24,8 @@ lrlex_mod!("config.l");
 lrpar_mod!("config.y");
 
 pub struct Config {
+    /// If true, then daemonise (i.e. disassociate from the terminal and log errors via syslog).
```

Comment:
Because it says here that errors will go to syslog. Perhaps drop that part until the support is implemented?

Reply:
The support is implemented, but I plonked it in a separate PR. I'm inclined to leave this comment as-is -- it's a minor, and very temporary, wart.

Reply:
OK